components/faq/questions.js

```diff
@@ -0,0 +1,227 @@
+        ' May 2016, with retroactive status to 15 June 2015. Donations to the organization are' +
+        ' deductible as charitable contributions.',
+    },
+    {
+      question: 'Where is Operation Code based? Do you have a location near me?',
+      answer:
+        'Operation Code, much like software, is built from anywhere with an internet connection,' +
+        " and is not based in one location. While we're headquartered in Portland, the entire" +
+        ' organization is decentralized, including the board of directors and the core team. This' +
+        " allows us to more effectively serve the entire military community, whether they're" +
+        " veterans or military spouses, whether they're OCONUS or in-country. We have chapters" +
+        ' all over the nation. Use Slack chat and join the closest town to you!',
+    },
+    {
+      question: 'Who does Operation Code serve?',
+      answer:
+        "Operation Code serves our nation's finest who've worn the uniform and their families" +
+        ' who are interested in coding and software development. Our programs are offered at no' +
+        ' cost to the military community, including veterans, transitioning service members, and' +
+        ' military spouses and families.',
+    },
+    {
+      question:
+        'What do I need to start learning software development and how much is this going to cost?',
+      answer:
+        'First off, you will need access to a solid computer, a good browser (ex. Chrome), and a' +
+        ' strong internet connection. It costs nothing to start learning to code and receive' +
+        ' software mentorship through Operation Code, you just need a positive attitude,' +
+        ' persistence and grit, and a thirst for new knowledge. We have over a dozen channels,' +
+        ' from Ruby to JavaScript, iOS to Android, and from Free Code Camp to edX study groups.',
+    },
+    {
+      question: 'What is available to start learning to code today?',
+      answer: (
+        <>
+          Our friends at the New York City-based, The Flatiron School created{' '}
+          <OutBoundLink href="https://learn.co/">Learn.co</OutBoundLink>, an online platform to get
+          introduced to web development and the popular web framework, Ruby on Rails. Request an
+          invite <OutBoundLink href="https://learn.co/join/operation-code">here</OutBoundLink> and
+          then join the #learn-dot-co channel in our Slack. Another resource is{' '}
+          <OutBoundLink href="https://www.learnhowtoprogram.com/courses">
+            learnhowtoprogram.com
+          </OutBoundLink>
+          , a resource maintained by Epicodus.
+        </>
+      ),
+    },
+    {
+      question: 'What are the hours of operation for Operation Code?',
+      answer: (
+        <>
+          Operation Code is different in that we don
+          {"'"}t have regular business office hours. The team can usually be found in our{' '}
```

Comment:
can you use HTML entities instead like `&apos;` and `&quot;`

https://www.w3schools.com/html/html_entities.asp

Reply:
Yep, thanks for the link!